### DISCUSSION TITLE
Issues with sfdx-git-delta detection of Custom Label Changes

### DISCUSSION THREAD

**Discussion by kusalgrg** [scolladon/sfdx-git-delta#689] [Q&A]
I'm encountering an issue with 'sfdx-git-delta' plugin when using the **sfdx sgd:source:delta --to HEAD --from HEAD~1 --source src --generate-delta** command. It appears that the plugin is not detecting changes to custom labels in my Salesforce DX project, but it is picking up other metadata changes. Can someone help me troubleshoot this problem?

In the past, custom labels were detected by the plugin. I have verified that I am using the latest version of the sfdx-git-delta plugin v5.25.2. Additionally, I ran the command with an older version of the plugin, but it still did not detect any changes.

**Steps Taken:**
I have tried the following steps to resolve the issue:
- Checked my repository to ensure that the custom label changes are present in the commit specified (HEAD~1 to HEAD).
- Verified that the 'sfdx-git-delta' is installed and up-to-date.
- Checked for any errors or warnings in the command output, but there are no specific messages related to the custom label changes.
- I made changes to custom labels by updating their values. These changes are reflected in the Git commit history.

**Versions:**
I am currently using sfdx version **sfdx-cli/7.206.6 win32-x64 node-v18.15.0** and **sfdx-git-delta v5.25.2**. If there are any known issues or compatibility concerns with specific versions, please let me know.

Any help or insights into resolving this issue would be greatly appreciated!

**Comment by scolladon** [comment]
Hi @kusalgrg!

Thanks for raising this issue and thanks for contributing in making this project better!
I am sorry to hear that you have issue with custom labels...

Would it be ok if I convert this discussion into an issue ? 
Or do you prefer create the issue yourself, so you can follow our template and provide us all the information we need ? (Do not forget steps to reproduce the issue, so I can run it on my local machine)

**Reply by kusalgrg** [reply]
Hi @scolladon,

I've opened an issue https://github.com/scolladon/sfdx-git-delta/issues/693.
